### ISSUE TITLE
[arm32][5.0] Cannot invoke initializer for type 'NSRect' with an argument list of type '(x: Float, y: Float, width: Float, height: Float)

### ISSUE THREAD

**Issue by uraimo** [uraimo/buildSwiftOnARM#39]
No workaround afaik, haven't looked into the specific error yet either.

```
/mnt/buildSwiftOnARM/swift-corelibs-foundation/Foundation/NSGeometry.swift:689:12: error: cannot invoke initializer for type 'NSRect' with an argument list of type '(x: Float, y: Float, width: Float, height: Float)'
    return NSRect(x: x, y: y, width: w, height: h)
           ^
/mnt/buildSwiftOnARM/swift-corelibs-foundation/Foundation/NSGeometry.swift:689:12: note: overloads for 'NSRect' exist with these partially matching parameter lists: (x: CGFloat, y: CGFloat, width: CGFloat, height: CGFloat), (x: Double, y: Double, width: Double, height: Double), (x: Int, y: Int, width: Int, height: Int)
    return NSRect(x: x, y: y, width: w, height: h)
           ^
/mnt/buildSwiftOnARM/swift-corelibs-foundation/Foundation/CGFloat.swift:477:16: warning: 'Hashable.hashValue' is deprecated as a protocol requirement; conform type 'CGFloat' to 'Hashable' by implementing 'hash(into:)' instead
    public var hashValue: Int {
               ^
[87/176] Generating Foundation.dir/NSData.swift.o, Foundation.dir/NSData.swift.swiftmodule, Foundation.dir/NSData.swift.swiftdoc
/mnt/buildSwiftOnARM/swift-corelibs-foundation/Foundation/NSData.swift:323:22: warning: 'withUnsafeBytes' is deprecated: use `withUnsafeBytes<R>(_: (UnsafeRawBufferPointer) throws -> R) rethrows -> R` instead
        return other.withUnsafeBytes { (bytes2: UnsafePointer<UInt8>) -> Bool in
                     ^
/mnt/buildSwiftOnARM/swift-corelibs-foundation/Foundation/NSRange.swift:261:43: warning: redundant same-type constraint 'S.Index' == 'String.Index'
        where R.Bound == S.Index, S.Index == String.Index {
                                          ^
/mnt/buildSwiftOnARM/swift-corelibs-foundation/Foundation/NSRange.swift:260:40: note: same-type constraint 'S.Index' == 'String.Index' implied here
    public init<R: RangeExpression, S: StringProtocol>(_ region: R, in target: S)
                                       ^
/mnt/buildSwiftOnARM/swift-corelibs-foundation/Foundation/NSData.swift:1015:15: warning: 'withUnsafeBytes' is deprecated: use `withUnsafeBytes<R>(_: (UnsafeRawBufferPointer) throws -> R) rethrows -> R` instead
        other.withUnsafeBytes {
              ^
/mnt/buildSwiftOnARM/swift-corelibs-foundation/Foundation/NSData.swift:1039:14: warning: 'withUnsafeBytes' is deprecated: use `withUnsafeBytes<R>(_: (UnsafeRawBufferPointer) throws -> R) rethrows -> R` instead
        data.withUnsafeBytes {
             ^
ninja: build stopped: subcommand failed.
./swift/utils/build-script: fatal error: command terminated with a non-zero exit status 1, aborting
```

(Obvious low effort workaround: Casting)

**Comment by uraimo** [comment]
See [PR#2226](https://github.com/apple/swift-corelibs-foundation/pull/2226)

**Comment by uraimo** [comment]
Merged in 5.1
